pkg/appgateserver/config/errors.go

```diff
@@ -0,0 +1,11 @@
+package config
+
+import sdkerrors "cosmossdk.io/errors"
+
+var (
+	codespace                                = "appgate_config"
+	ErrAppGateConfigUnmarshalYAML            = sdkerrors.Register(codespace, 1, "config reader cannot unmarshal yaml content")
+	ErrAppGateConfigEmptySigningKey          = sdkerrors.Register(codespace, 2, "empty signing key in app gate config")
```

Comment:
```suggestion
	ErrAppGateConfigEmptySigningKey          = sdkerrors.Register(codespace, 2, "empty signing key in appgate config")
```

Please stay consistent. I don't think it matters if we use `appgate` or `app gate` or `AppGate`, etc, as long as we're consistent.